test/e2e/pipeline/infra/cluster-api/setup.sh

```diff
@@ -100,6 +100,11 @@ do
   # Install metrics-server
   "${KUBECTL}" apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.6.4/components.yaml --kubeconfig "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
   "${KUBECTL}" -n kube-system patch deployment metrics-server --type json --patch '[{"op":"add","path":"/spec/template/spec/containers/0/args/-","value":"--kubelet-insecure-tls"}]' --kubeconfig "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
+
+  # Install kyverno for network tests
+  "${HELM}" repo add kyverno https://kyverno.github.io/kyverno/
+  "${HELM}" repo update
+  "${HELM}" install kyverno kyverno/kyverno -n kyverno --create-namespace --kubeconfig "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
```

Comment:
Are kyverno images pulled from docker? If that, can we pull them from the docker proxy?

Reply:
Of course